velox/exec/HashProbe.h

```diff
@@ -55,7 +87,26 @@ class HashProbe : public Operator {
   void clearDynamicFilters() override;
 
  private:
-  // Sets up 'filter_' and related members.
+  void setState(State state);
+  void stateTransitionCheck(State state);
```

Comment:
nit: rename to checkStateTransition and make it const to be consistent with checkRunning() method below